precommit/mirsg-hooks.yaml

```
@@ -12,10 +12,10 @@ repos:
               rules: {
                 anchors: enable,
                 braces: {
-                  forbid: true
+                  forbid: non-empty
                 },
                 brackets: {
-                  forbid: true
+                  forbid: non-empty
```

From a readability point of view, I actually think it is better to remove them altogether
```suggestion
                  forbid: true
```